root/communication-rmi/src/main/java/at/fhv/itb5c/rmi/server/rmiclasses/TournamentRMI.java

```diff
@@ -19,6 +20,9 @@
 
 	public TournamentRMI() throws RemoteException {
 		super();
+		_homeTeams = new HashSet<>();
+		_guestTeams = new HashSet<>();
+		_matches = new HashSet<>();
```

Comment:
These variables should rather be initialized in the entity class. For example, see Team and TeamDTO.